core/zip/contrib/amd64/crc32-pclmul_asm.S

```diff
@@ -0,0 +1,258 @@
+/* GPL HEADER START
+ *
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 only,
```

Comment:
AFAIK LGPL 2.1+ code cannot include GPL2 code. We cannot prevent people from providing it themselves when building ROOT, but we cannot include it.

Reply:
@bbockelm do you know how it possibly could be done?